ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -131,28 +131,46 @@ def aws_cleanup():
         required=False,
         help="The name of the AWS region to delete the resources from"
     )
+    parser.add_argument(
+        '--force',
+        action='store_true',
+        required=False,
+        help="Skip confirmation prompt"
+    )
+    parser.add_argument(
+        '--skip-prefixes',
+        action='store_true',
+        required=False,
+        help="Skip prompt for additional prefixes to spare"
+    )
     logging.basicConfig(level=logging.DEBUG)
     args = parser.parse_args()
 
-    confirmation = input(
-        'Careful! This action could be highly destructive. Are you sure you want to proceed? '
-    )
-    prefixes_hours = input(
-        "Press Enter if there are no cluster prefixes to spare.\n"
-        "If you would like the cleanup to spare specific cluster prefixes, "
-        "please enter them along with the time allowed for these to be kept "
-        "running, in a dictionary representation.\nAn example: "
-        "{\'prefix1\': 36, \'prefix2\': 48}\" "
-    )
-    assert confirmation == defaults.CONFIRMATION_ANSWER, "Wrong confirmation answer. Exiting"
+    if not args.force:
+        confirmation = input(
+            'Careful! This action could be highly destructive. '
+            'Are you sure you want to proceed? '
+        )
+        assert confirmation == defaults.CONFIRMATION_ANSWER, (
+            "Wrong confirmation answer. Exiting"
```

Comment:
is it good to display what is the confirmation answer. By seeing the Question, I prefer "y" or "yes" but defaults.CONFIRMATION_ANSWER is different than what I think. 
WDYT?

Reply:
Coady is just indenting this in the new condition for force param. Doesn't change the logic here. If there is issue with this it should be addressed in different PR. You can open the issue for this.